nrf_security/doc/backend_config.rst

```diff
@@ -190,49 +172,27 @@ Feature support
 |             |                   | 256-bit key |                       |
 +-------------+-------------------+-------------+-----------------------+
 
+.. note:
```

Comment:
With a simple sentence we can make this entire table and many of the other tables unnecessary.

"Supported key sizes are 128-bit, 192-bit and 256-bit, except for cc310 which only supports 128-bit."

Reply:
The change will be done in a later improvement. Won't fix this now